### DISCUSSION TITLE
Masks vs Zones - How to exclude an entire zone?

### DISCUSSION THREAD

**Discussion by KevinPeters80** [blakeblackshear/frigate#4356] [Ask A Question]
I live on a busy-ish street and am not interested on the cars & people on the street at the end of my driveway. Initially I masked out the street, but now I have come to understand that this is not the correct way to do it and that I should use Zones. Therefore I have created a Street_Zone but now I want to exclude it from triggering events. 

I know there is a required_zones property, but how can I exclude an entire zone?

**Comment by NickM-27** [comment]
Do you only have a street zone? This is the opposite of how you're supposed to do it, you're supposed to create zones for areas of interest not areas with a lack of interest.

**Comment by KevinPeters80** [comment]
Hi Nick, I have the following zones:
Neighbour_left
Neighbour_right
Driveway
Driveway_Entrance
Street

I am interested in all except the street, so I guess I should just delete the Street Zone and then set the the rest of them to be required_zones? The only problem is that this is conflicting with my other question about stopping it alerting about a stationary car on Driveway...

Thanks for your help

**Reply by NickM-27** [reply]
There's no need to delete the street zone, just add the zones you want to create events and don't put the others in the list
